### PR TITLE
Added stars and forks 

### PR DESCRIPTION
**Pull request by AK-007**
Issue Number #176 

Link : https://ak-007.github.io/OpenCode-Collaborative/

### CODE REVIEW

File: contestants.json [hunk 1]
```diff
@@ -226,9 +226,10 @@
          "name":"Ayush Gupta",
          "imageurl":"contestant/ayush.png",
          "about":"Consectatio Excellentiae",
-         "college":"",
+         "college":"Birla Institute of Technology, Mesra",
          "facebook":"https://www.facebook.com/ayush926",
          "github":"https://github.com/AK-007"
```
Comment (pradeepgangwar) [inline contestants.json]: use `,` at the end of the line.

File: js/apicall.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+   
+  var url="https://api.github.com/orgs/opencode18/repos?callback"; 
+  $.getJSON(url,function(data)
+  {
+    var stars=data[3].watchers;
+    var forks=data[3].forks;
+    $('#star').text(stars);
+    $('#fork').text(forks);
+  });
+})(jQuery);
```
Comment (pradeepgangwar) [inline js/apicall.js]: leave a line at the end of file.

File: css/style.css [hunk 3]
```diff
@@ -252,6 +252,39 @@ header .card-underline{
...
+  left: 50%;
+  transform: translate(-100%,0%);
+  margin-right: 20px;
+}
+.fo
+{
+  position: absolute;
+  left: 50%;
+  margin-left: 20px;
+}
```
Comment (pradeepgangwar) [inline css/style.css]: Design looked good. :+1:

**Comment by Anupam-dagar** [comment]
@aswinzz better put them in nav bar?

**Comment by AK-007** [comment]
@Anupam-dagar should i add it in the right corner of nav bar.

**Comment by Anupam-dagar** [comment]
yeah, comment a screenshot here, keep the design same for buttons

**Comment by pradeepgangwar** [comment]
Try it ... we can see that in Link once you change. moreover there are conflicts fix them.

**Comment by AK-007** [comment]
Help me fix conflicts,how can I do it?

**Comment by pradeepgangwar** [comment]
Conflicts occur when changes in your file and those on master of this repo do not match or you change some line and that line on original file has some other content than what is with you. Just pull recent changes from master of this repo and check which one to keep and what to discard. Read this https://confluence.atlassian.com/bitbucket/resolve-merge-conflicts-704414003.html and some other online resource for more help.

**Comment by AK-007** [comment]
@aswinzz @Anupam-dagar @pradeepgangwar Check it out
screenshot 60

**Comment by pradeepgangwar** [comment]
@AK-007 IMO, Shift it to right little bit more.

**Comment by AK-007** [comment]
screenshot 61

**Comment by pradeepgangwar** [comment]
@AK-007 Looks ok ... add commit with this one and share live link

**Comment by AK-007** [comment]
Link : https://ak-007.github.io/OpenCode-Collaborative/

**Comment by Anupam-dagar** [comment]
LGTM too, please resolve merge conflicts

**Comment by aswanthkoleri** [comment]
@AK-007 Please resolve the conflict.Otherwise its a good PR

**Comment by AK-007** [comment]
@Anupam-dagar It is showing "These conflicts are too complex to resolve in the web editor".

**Comment by pradeepgangwar** [comment]
Solve in local editor

**Comment by AK-007** [comment]
@pradeepgangwar I am creating new PR.

**Pull request closed without merging**